projects/packages/publicize/changelog/update-list-publicize-connections

```diff
@@ -0,0 +1,4 @@
+Significance: minor
```

Comment:
Is this changelog needed here? It doesn't look like we're updating anything that would affect the package